embassy-nrf/Cargo.toml

```diff
@@ -107,6 +117,7 @@ _nrf5340-app = ["_nrf5340", "nrf5340-app-pac"]
 _nrf5340-net = ["_nrf5340", "nrf5340-net-pac"]
 _nrf5340 = ["_gpio-p1", "_dppi"]
 _nrf9160 = ["nrf9160-pac", "_dppi"]
+_nrf9120 = ["nrf9120-pac", "_dppi"]
```

Comment:
I think if you define a `_nrf91` feature you could simplify a few of the cfg's in the crate. Like, "if nrf91" instead of "if nrf9160 or nrf9120"

Reply:
I thought the same.  Let me look into that.  I have my 9160dk back so I can test against both.